bigchaindb/common/schema/vote.yaml

```diff
@@ -0,0 +1,69 @@
+---
+"$schema": "http://json-schema.org/draft-04/schema#"
+id: "http://www.bigchaindb.com/schema/vote.json"
+type: object
+additionalProperties: false
+title: Vote Schema
+description: |
+    A Vote is an endorsement of a Block (identified by a hash) by
+    a node (identified by a public key).
+
+    The outer Vote object contains the details of the vote being made
+    as well as the signature and identifying information of the node
+    passing the vote.
+required:
+- node_pubkey
+- signature
+- vote
+properties:
+  node_pubkey:
+    type: "string"
+    pattern: "[1-9a-zA-Z^OIl]{43,44}"
+    description: |
+        Ed25519 public key identifying the voting node.
+  signature:
+    type: "string"
+    pattern: "[1-9a-zA-Z^OIl]{86,88}"
+    description:
+        Ed25519 signature of the ``vote`` object.
```

Comment:
This sentence doesn't make it clear which object is the "vote object." Is it the outer one or the inner one? I think it's the inner one, the one you also call the "Vote Details".